go.graphics.swing/src/main/java/go/graphics/swing/text/JOGLTextDrawer.java

```diff
@@ -20,31 +20,43 @@
 import go.graphics.text.TextDrawer;
 
 import java.awt.Font;
+import java.awt.GraphicsDevice;
+import java.awt.GraphicsEnvironment;
 import java.awt.geom.Rectangle2D;
 
 /**
  * This class is a text drawer used to wrap the text renderer.
- * 
+ *
  * @author michael
  */
 public final class JOGLTextDrawer implements TextDrawer {
 
-	private static final String FONTNAME = "Arial";
+	private static final String FONT_NAME = "Arial";
+
+	private static final int DEFAULT_DPI_WIDTH = 1920;
+	private static final float SCALING_FACTOR;
+
+	static {
+		GraphicsDevice graphicsDevice = GraphicsEnvironment.getLocalGraphicsEnvironment().getDefaultScreenDevice();
+		int screenWidth = graphicsDevice.getDisplayMode().getWidth();
```

Comment:
Are you sure computing this in advance is a good idea? I think it should depend on the main viewport size and be re-computed after the window size is changed.